src/testCommon/java/uk/gov/hmcts/darts/test/common/data/TranscriptionWorkflowTestData.java

```diff
@@ -36,10 +31,27 @@ public static TranscriptionWorkflowEntity workflowForTranscription(Transcription
         return transcriptionWorkflow;
     }
 
-    public static TranscriptionWorkflowEntity workflowForTranscriptionWithStatus(TranscriptionEntity transcription, TranscriptionStatusEnum status) {
+    public TranscriptionWorkflowEntity workflowForTranscriptionWithStatus(TranscriptionEntity transcription, TranscriptionStatusEnum status) {
         var transcriptionWorkflow = workflowForTranscription(transcription);
         transcriptionWorkflow.setTranscriptionStatus(new TranscriptionStatusEntity(status.getId()));
         return transcriptionWorkflow;
     }
 
-}
+    @Override
+    public CustomTranscriptionWorkflowEntity.CustomTranscriptionWorkflowEntityBuilderRetrieve someMinimal() {
+        CustomTranscriptionWorkflowEntity.CustomTranscriptionWorkflowEntityBuilderRetrieve retrieve =
+            new CustomTranscriptionWorkflowEntity.CustomTranscriptionWorkflowEntityBuilderRetrieve();
+
+        retrieve.getBuilder().transcription(PersistableFactory.getTranscriptionTestData().minimalTranscription())
+            .workflowActor(minimalUserAccount())
+            .transcriptionStatus(new TranscriptionStatusEntity(TranscriptionStatusEnum.REQUESTED.getId()))
+            .workflowTimestamp(OffsetDateTime.now());
+
+        return retrieve;
+   }
+
+    @Override
+    public CustomTranscriptionWorkflowEntity.CustomTranscriptionWorkflowEntityBuilderRetrieve someMaximal() {
+        return someMinimal();
```

Comment:
Should be the max values of fields such as max length max size etc (Same for all maxImal's)

Reply:
See new Persistable default interface with associated unsupported exception. We can implement the maximum another time as tests do not depend upon it